src/Exceptionless.Core/Utility/SampleDataService.cs

```diff
@@ -65,8 +77,12 @@ public class SampleDataService {
             _billingManager.ApplyBillingPlan(organization, _billingPlans.UnlimitedPlan, user);
             organization = await _organizationRepository.AddAsync(organization, o => o.Cache()).AnyContext();
 
-            var project = new Project { Id = TEST_PROJECT_ID, Name = "Disintegrating Pistol", OrganizationId = organization.Id };
-            project.NextSummaryEndOfDayTicks = SystemClock.UtcNow.Date.AddDays(1).AddHours(1).Ticks;
+            var project = new Project {
+                Id = TEST_PROJECT_ID,
```

Comment:
@ejsmith is there any reason we used a second const id here instead of just using TestConstants?